storybook/src/admin/mui/Chip.tsx

```diff
@@ -8,538 +9,317 @@ function Story() {
         <Stack spacing={4}>
             <Card>
                 <CardContent>
-                    <Typography variant="h3" gutterBottom>
+                    <Typography variant="h3" mb={10}>
                         Chips
                     </Typography>
-
+                    <Box mb={10}>
+                        <Typography variant="h4" gutterBottom>
+                            Customizable Chip
+                        </Typography>
+                        <Typography gutterBottom variant="body2">
+                            Use the story knobs to customize the props of the Chip below.
+                        </Typography>
+                        <Chip
+                            label="Customizable Chip"
+                            color={select("Color", ["default", "primary", "secondary", "success", "error", "warning"], "default")}
+                            variant={select("Variant", ["filled", "outlined"], "filled")}
+                            size={select("Size", ["small", "medium"], "medium")}
+                            disabled={boolean("Disabled", false)}
+                            clickable={boolean("Clickable", false)}
+                            icon={boolean("Icon", false) ? <ChevronDown /> : undefined}
+                            onDelete={boolean("Deletable", false) ? () => console.log("Delete") : undefined}
+                        />
+                    </Box>
+                    <Box mb={10}>
```

Comment:
Why have this list of examples when we have the customizable chips?

Reply:
I think it can be helpful to have a good overview of different usages.